package.json

```
@@ -4,7 +4,8 @@
   "description": "https://omegasisters.github.io/homepage",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js"
+    "start": "node index.js",
+    "format": "prettier --write \"**/*.{css,html}\""
```

```suggestion
    "format": "prettier --write \"**/*.{css,html,json,md}\""
```

jsonやmarkdownも整形できると良さそうです！

---

おっと忘れてました！ありがとうございます！